massa-node/base_config/openrpc.json

```diff
@@ -394,12 +394,31 @@
                     "description": "Massa public api"
                 }
             ],
-            "params": [],
+            "params": [
+                {
+                    "name": "page_request",
```

Comment:
As it's designed to be reused, you could extract it into a component schema section and add it to contentDescriptors because it's an INPUT which is directly manipulated by the user.